media/js/firefox/new.js

```diff
@@ -27,6 +62,21 @@
                 // (from a download button) then we want them to see the same scene 2
                 // as non-firefox users and initiate a download
                 $html.addClass('firefox-latest');
+
+                uiTourGetConfiguration('appinfo', function(config) {
```

Comment:
I was on the fence about this. The check would gradually become unnecessary, so I was thinking it wouldn't be worth adding what would eventually amount to needless cycles. Then I realized this bit of code is running in the `<head>` before `doc.ready`, meaning the markup hasn't even been parsed yet, meaning I'd have to re-work the timing on this bit of code to wait until `doc.ready`, which I'd rather not do here.